xbmc/addons/kodi-dev-kit/include/kodi/c-api/gui/input/action_ids.h

```diff
@@ -512,6 +512,9 @@ enum ADDON_ACTION
   /// @brief <b>`164`</b>: Set rating.
   ADDON_ACTION_SET_RATING = 164,
 
+  /// @brief <b>`165`</b>: Switch to previous subtitle of movie. Can b used in videoFullScreen.xml window id=2005
```

Comment:
Because this gets rendered in our docs, I would suggest:

```
  /// @brief <b>`165`</b>: Switch to previous subtitle of movie. Can be used in
  /// the VideoFullScreen.xml window (with ID 2005).
```

Reply:
Existing code is written in different times under different conditions, always ask if it can be done better today.